release/pypi/promote_pypi_to_production.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+# Make sure to update these versions when doing a release first
+PYTORCH_VERSION=${PYTORCH_VERSION:-1.11.0}
```

Comment:
Do you want to update those to 1.12 already?

Reply:
Yup, should be updated in the latest